visivo/commands/compile.py

```diff
@@ -62,4 +10,11 @@ def compile(working_dir, output_dir, target):
     """
     Parses the files in your working directory, extracting visivo configurations and then using those configurations to build the trace queries and a project.json file in your target directory. Queries are not run on compile, just written.
     """
+    from visivo.logging.logger import Logger
+
+    Logger.instance().debug("Compiling")
+
+    from visivo.commands.compile_phase import compile_phase
```

Comment:
So a lot of the performance benefits are coming from moving imports into the command?

Reply:
Or it just allows the tool to say it's doing something sooner maybe?